core/src/main/scala/kafka/utils/KafkaScheduler.scala

```diff
@@ -121,7 +123,7 @@ class KafkaScheduler(val threads: Int,
   
   def isStarted: Boolean = {
     this synchronized {
-      executor != null
+      executor != null && !executor.isShutdown
```

Comment:
The `isShutdown` check is redundant right? i.e., since the shutdown and setting to null are in a synchronized block? Also, I think we should change the message in `ensureStarted` to say "Kafka scheduler is not running." - since it could have been either not started or may have been shut down.